site/en/configure/memory.md

```diff
@@ -1,22 +1,27 @@
 Project: /_project.yaml
 Book: /_book.yaml
 
-# Running Bazel with Limited RAM
+
+# Optimize Memory
 
 {% include "_buttons.html" %}
 
-This page describes how to use flags to run Bazel with limited RAM.
+This page describes how to limit and reduce the memory Bazel uses.
+
+## Running Bazel with Limited RAM
 
 In certain situations, you may want Bazel to use minimal memory. You can set the
 maximum heap via the startup flag
-[`--host_jvm_args`](/docs/user-manual#host-jvm-args),
+[`--host_jvm_args`](/user-manual#host-jvm-args),
```

Comment:
I assume you wanted to link to `/reference/command-line-reference#flag--host_jvm_args`

Reply:
See other comment re user manual vs command line ref.